myJava.js

```diff
@@ -0,0 +1,95 @@
+function myFunction(){
+		var verb = document.getElementById("verbs").value;
+		var noun = document.getElementById("nouns").value;
+		var adverb = document.getElementById("adverbs").value;
+		var adjective = document.getElementById("adjectives").value;
+
+		var verbArr = verb.split(",");
+		var nounArr = noun.split(",");
+		var adverbArr = adverb.split(",");
+		var adjectiveArr = adjective.split(",");
+
+		function makeBold(arr){
+			return arr.map(function (item){
```

Comment:
This array.map function is really cool! I want you to talk to the class about what it is and why you used it in this instance.